pyramid_nacl_session/tests/test_session.py

```diff
@@ -1,34 +1,115 @@
-from pyramid import testing
+# -*- coding: utf-8 -*-
+
 import unittest
 import webtest
+from pyramid import testing
+
+
+class SomeClass:
```

Comment:
Given that we are still supporting Python 2.7, use `object` for clarity:

```suggestion
class SomeClass(object):
```